site/content/docs/snippets/_index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Snippets Overview
+weight: 70
+slug: /
+aliases:
+- "/docs/snippets"
+- "/docs/browser-checks/partials-code-snippets/"
+
+menu:
+  docs:
+    parent: "Snippets"
+---
+
+Snippets are JavaScript files which can be reused across checks. Snippets can be used in [Browser checks](/docs/browser-checks/), as well as [setup and teardown scripts](/docs/api-checks/setup-teardown-scripts/) for API checks. By reusing code with snippets, checks can be made more maintainable and easier to develop.
```

Comment:
```suggestion
Snippets are JavaScript files which can be reused across [Browser checks](/docs/browser-checks/), as well as [setup and teardown scripts](/docs/api-checks/setup-teardown-scripts/) for API checks. By reusing code with snippets, checks can be made more maintainable and easier to develop.
```

Reply:
Less words are always better words. ;)

Reply:
Thanks for the proposal. You're right, that's better :)